src/rHelpPanel.ts

```diff
@@ -395,11 +396,12 @@ export class HelpPanel implements api.HelpPanel {
 	private getWebview(): Webview {
 		// create webview if necessary
 		if(!this.panel){
-			const webViewOptions: WebviewOptions = {
-				enableScripts: true,
+			const webViewOptions: WebviewOptions & WebviewPanelOptions = {
+        enableScripts: true,
+        enableFindWidget: true
 			};
-			this.panel = window.createWebviewPanel('rhelp', 'R Help', this.viewColumn, webViewOptions);
-
+      this.panel = window.createWebviewPanel('rhelp', 'R Help', this.viewColumn, webViewOptions);
+      
 			// virtual uris used to access local files
```

Comment:
Please fix the indentation here :)

Reply:
done (47b869a9ce519aaa893a6365a54ec27a13093ff2)

Reply:
Is there a reason why space- and tab-indentation is mixed in the project? Why not stick with one style (preferring spaces)?

Reply:
I'd assume that is just the result of the different defaults/preferences from different developers. This could probably be fixed by some linter rule